### DISCUSSION TITLE
can you use symlinks in minio?

### DISCUSSION THREAD

**Discussion by vinder11** [minio/minio#15412] [Q&A] [answered]
Can Minio work with symlinks from folders that are on the same server but are outside of the main folder where Minio saves the files?

Example
root
```
  |---minio
  |      |
  |      |---.mini.sys
  |      |---bucket-one
  |              |---pdf
  |              |---documents
  |              |---images (this folder is a symbolic link of another folder that is at the level of the minio folder)
  |      |---bucket-two
  |---images (folder original that I want to make the symbolic link)
  |      |---image-one.png
  |      |---image-two.png
  |      |---image-three.png
```

Right now I have seen that in the Minio console it shows the folder that is a symbolic link but when I enter the folder it does not show me its content of the folder that is linked or when I want to upload a file from the console to the folder that is a symbolic link, I get the access denied message.
Is it possible to work inside Minio with folders that are symbolic links? Is there any documentation about it?

**Comment by harshavardhana** [comment, marked as answer]
No it is not allowed

**Reply by vinder11** [reply]
Thank you very much for your answer 😃👍

**Reply by Cherrymelon** [reply]
> No it is not allowed

It has any plan to support this feature? @harshavardhana

**Comment by Cherrymelon** [comment]
It has any plan to support this feature?

**Reply by klauspost** [reply]
No.
